tests/test_base.py

```diff
@@ -423,6 +423,31 @@ def test_environ_overrides_default(self):
         with Environ(TESTNS_SECTION1_FOO='blah'):
             self.assertEqual('blah', getter.getstr('section1.foo'))
 
+    def test_config_to_INI(self):
+        getter = getconf.ConfigGetter('TESTNS')
+        getter.getstr('section1.foo', default='bar', doc='Foo\nbar')
+        getter.getlist('section2.foo', default=['first', 'second'])
+        getter.getlist('section2.bar', default=[], doc='Emptylist')
+        getter.getbool('section3.bar', default=True, doc='Important bool value')
+
+        self.assertEqual(getter.to_INI(), """\
+[section1]
+; type=str - Foo bar
+;foo = bar
+
+[section2]
+; type=list - Emptylist
+;bar =
+; type=list
+;foo =
+;  first
+;  second
+
+[section3]
+; type=bool - Important bool value
+;bar = 1
+""")
+
```

Comment:
Could we add a test that takes the generated output, uncomments the values, and tries to parse it again?